components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -45,6 +45,7 @@ type FileSystem struct {
 	MountLocation string `json:"mount_location"`
 }
 type ObjectStorage struct {
+	Profile    string `json:"profile"`
```

Comment:
Profile is not a part of ObjectStorage struct. Should keep it outside

Reply:
Changed Location to Config models